_maps/RandomRuins/SpaceRuins/spacehotel.dmm

```diff
@@ -396,10 +434,40 @@
 /obj/machinery/atmospherics/pipe/smart/manifold4w/scrubbers/hidden/layer2,
 /turf/open/floor/plating,
 /area/ruin/space/has_grav/hotel)
+"dR" = (
+/obj/structure/chair/sofa/right,
+/obj/item/bedsheet/patriot{
+	pixel_y = 3;
+	pixel_x = 6
+	},
+/obj/structure/window/reinforced/survival_pod/spawner/directional/north,
+/obj/structure/window/reinforced/survival_pod/spawner/directional/west,
+/obj/effect/turf_decal/siding/dark{
+	dir = 9
+	},
+/turf/open/floor/carpet/blue,
+/area/ruin/space/has_grav/hotel/security)
 "eb" = (
 /obj/machinery/light/directional/south,
 /turf/open/floor/iron/grimy,
 /area/ruin/space/has_grav/hotel)
+"ee" = (
+/obj/machinery/camera{
+	c_tag = "Medbay Lobby";
```

Comment:
why is there a medbay lobby camera?

Reply:
Yeah I can't actually find that but there doesn't actually seem to be one, shruggers?